### ISSUE TITLE
Reuse pretrain coco and add my dataset

### ISSUE THREAD

**Issue by ductienuit** [qqwweee/keras-yolo3#533]
Hi guy, I already test. The result car, truck, bus in coco pretrain is good for me, but in some scene it can't detect. So I want to add some data with my scene, reuse pretrain coco to transfer learning but yolov3 have no softmax to feature extractor. Which soIution is possible between train model from scratch or feature extractor ? If It's feature extractor, which layer need to freeze. I just want to learn from another people here.

**Comment by smalldroid** [comment]
I have problem same as you. Does anyone have experience to train on custom data set (rip 3k image from coco for each classes) with custom classes? I trained 9 class, with pretrained weight yolov3 on coco dataset but its detection is not good.
